pom.xml

```diff
@@ -64,6 +64,10 @@
             </exclusion>
         </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-validation</artifactId>
+    </dependency>
```

Comment:
See https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-2.3-Release-Notes#validation-starter-no-longer-included-in-web-starters

Without this dependency, I'm unable to successfully run `./mvnw package`